class/defaults.yml

```diff
@@ -1,3 +1,14 @@
 parameters:
   prom_label_proxy:
+    multi_instance: true
     namespace: syn-prom-label-proxy
+    label: namespace
+    upstream: http://prometheus-k8s.monitoring.svc:9090
+    ingress:
+      annotations: {}
+      host: null
+    images:
+      prom-label-proxy:
+        registry: quay.io
+        image: prometheuscommunity/prom-label-proxy
+        tag: master@sha256:3e5d58c6163626df671022c2d44154bc9198ad0b42f00de7814b3ab622ff6636
```

Comment:
Do you know if there is also the intention to rename branch from master to main?

Reply:
No that I'm aware of. But I plan to switch to a release tag once there's a release.